lib/travis/yaml/doc/helper/keys.rb

```diff
@@ -33,6 +33,7 @@ def self.split(str)
           pip
           prose
           sdk
+          slack
```

Comment:
should this be `slack` or `stack`?

Reply:
Yeah weirdly it's actually correct. This stops us doing a dictionary lookup on the word `slack`, because it would return `stack` and we don't want to incorrectly assume this is a typo and substitute these words for each other.